src/fsae_firmware/README.md

```diff
@@ -0,0 +1,29 @@
+
+# Firmware Execution Instructions #
+
+These steps will explain how to deploy and execute new code on the teeny using the Arduino IDE, as well as integrate it with the Jetson.
+
+## Uploading Code to the Teensy ##
+
+Prereqs:
+	Arduino IDE (https://www.arduino.cc/en/Main/Software)
+	Teensy Loader (https://www.pjrc.com/teensy/loader.html)
```

Comment:
Need to install Teensyduino itself too

Reply:
Will add that in as well as links to the installation instructions.